src/main.py

```diff
@@ -47,8 +47,8 @@
     Config.set('graphics', 'width', width)
     Config.set('graphics', 'height', height)
 else:
-    Config.set('graphics', 'width', '800')
-    Config.set('graphics', 'height', '480')
+    Config.set('graphics', 'width', '1280')
+    Config.set('graphics', 'height', '800')
```

Comment:
I would change this back

Reply:
Left it for ease of testing for now, but yeah good point.

Reply:
Please change back :)